src/app/src/main/kotlin/org/vaccineimpact/api/app/app_start/route_config/OneTimeLinkRouteConfig.kt

```diff
@@ -10,6 +11,11 @@ object OneTimeLinkRouteConfig : RouteConfig
     val url = "/onetime_link/:token/"
 
     override val endpoints = listOf(
+            // This gets new style onetime tokens (like the reporting API)
+            Endpoint("/onetime_token/", controller, "getToken")
```

Comment:
didn't we at some point move over all underscores in endpoints to hyphens?

Reply:
As discussed, I've opened https://vimc.myjetbrains.com/youtrack/issue/VIMC-1440 to do this properly.